pkg/operator/ceph/object/bucket/util.go

```diff
@@ -125,12 +124,14 @@ func getService(c kubernetes.Interface, namespace, name string) (*v1.Service, er
 
 func randomString(n int) string {
 
-	r := rand.New(rand.NewSource(time.Now().UnixNano()))
-	var letterRunes = []rune("0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
-
-	b := make([]rune, n)
-	for i := range b {
-		b[i] = letterRunes[r.Intn(len(letterRunes))]
+	var letterRunes = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	var b = make([]byte, n)
+	if _, err := rand.Read(b); err != nil {
+		return ""
+	}
+	for k, v := range b {
+		b[k] = letterRunes[v%byte(len(letterRunes))]
 	}
-	return string(b)
+
+	return (string(b))
```

Comment:
shouldn't need parens
```suggestion
	return string(b)
```

Reply:
actually this g404 error code so done in a separate commit.